js/apps/admin-ui/src/clients/authorization/PermissionDetails.tsx

```diff
@@ -286,6 +286,7 @@ export default function PermissionDetails() {
                   {...register("resourceType", {
                     required: permissionType === "scope",
                   })}
+                  aria-label={t("resourceType")}
```

Comment:
We should not be adding an `aria-label` attribute here, as there is already a label in the `FormGroup`. It looks like the `fieldId` prop is set to the wrong input, which is why it creates accessibility issues.